cmd/remote-storage/app/server_test.go

```diff
@@ -51,14 +52,16 @@ func TestNewServer_CreateStorageErrors(t *testing.T) {
 	factory.On("CreateSpanWriter").Return(nil, nil)
 	factory.On("CreateDependencyReader").Return(nil, errors.New("no deps")).Once()
 	factory.On("CreateDependencyReader").Return(nil, nil)
-
+	telset := telemetery.Setting{
+		Logger:       zap.NewNop(),
+		ReportStatus: telemetery.HCAdapter(healthcheck.New()),
```

Comment:
if you are not using status reporter, don't pass real function
```suggestion
		ReportStatus: func(*component.StatusEvent) {},
```